owasm/src/ext/crypto/mod.rs

```diff
@@ -1,3 +1,4 @@
+pub mod alphavantage;
```

Comment:
Why is this in `crypto`, but the yahoo thing is on a different folder? Shouldn't this be in the other folder too since it's not intended for crypto?